aptos-move/vm-genesis/src/lib.rs

```diff
@@ -148,6 +153,37 @@ pub fn encode_genesis_change_set(
     change_set
 }
 
+fn validate_genesis_config(genesis_config: &GenesisConfigurations) {
+    assert!(
+        genesis_config.min_stake <= genesis_config.max_stake,
+        "Min stake must be smaller than or equal to max stake"
+    );
```

Comment:
I think we should probably move away from the assert system and move to Result returning since it's used by a CLI tool, but we can get back to that later.